models/Participant.model.js

```diff
@@ -5,12 +5,10 @@ const participantSchema = Schema({
   owner: {
     ref: "User",
     type: Schema.ObjectId,
-    required: true,
   },
   petition: {
     ref: "Petition",
     type: Schema.ObjectId,
-    required: true,
   },
```

Comment:
Did you encounter funky behavior with `required: true` here? I would think that `Participants` do need to be associated with an `Owner` as well as a `Petition`.

There might be something I've missed here though.